run-pr-git-repo-size

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash -ex
```

Comment:
this check should be part of https://github.com/cms-sw/cms-bot/blob/master/pr_testing/test_multiple_prs.sh#L259